kubectl-minio/cmd/tenant-expand.go

```diff
@@ -69,6 +69,7 @@ func newTenantExpandCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 	}
 	cmd = helpers.DisableHelp(cmd)
 	f := cmd.Flags()
+	f.StringVarP(&v.tenantOpts.NS, "namespace", "n", "", "k8s namespace for this MinIO tenant")
```

Comment:
should we also add `cmd.MarkFlagRequired("namespace")` ?

Reply:
not sure, current logic tries to get the namespace based on the tenant name if no namespace is provided, i believe that handling namespace as optional should be fine for now